crates/pixi-build-python/src/config.rs

```diff
@@ -32,6 +35,19 @@ impl PythonBackendConfig {
         self.noarch.unwrap_or(true)
     }
 
+    pub fn pip_config_string(&self) -> String {
+        if self.pip_config.is_empty() {
+            String::new()
+        } else {
+            self.pip_config
+                .iter()
+                .map(|s| format!("-C{}", s))
```

Comment:
```suggestion
                .map(|s| format!("--config-settings{}", s))
```
I like to use the full name in these cases

Reply:
Also isn't it safer to add a space?

Reply:
I switched to `--config-settings={input}`